src/lib/hooks/useSideMenu.js

```diff
@@ -132,18 +131,16 @@ export default (props = {}) => {
       },
     ]
 
-    if (!isWebApp) {
-      items.push({
-        icon: 'logout',
-        name: 'Logout',
-        action: () => {
-          fireEvent(LOGOUT)
-          AsyncStorage.clear()
-          window.location = '/'
-          slideOut()
-        },
-      })
-    }
+    items.push({
```

Comment:
It can be moved directly to the items array, no need to push as the condition is removed.

Reply:
removed